src/asset-swapper/utils/quote_info.ts

```diff
@@ -27,14 +27,16 @@ export function calculateQuoteInfo(params: {
     slippage: number;
 }): QuoteInfo {
     const { path, operation, assetFillAmount, gasPrice, gasSchedule, slippage } = params;
+    const { nativeOrders, bridgeOrders, twoHopOrders } = path.getOrdersByType();
+    const singleHopOrders = [...nativeOrders, ...bridgeOrders];
     // TODO: generalize calculateQuoteInfo to handle a mix multihop+multiplex.
 
     // NOTES: until multihop+multiplex is supported a path will have single hop order(s) xor a two hop order.
```

Comment:
"xor" or "or" ?

Reply:
exclusive or ( i guess all `or` is exclusive or in day-to-day language 🤣 )

(can't be both)